atomic_defi_design/Dex/Components/DexComboBox.qml

```diff
@@ -57,14 +57,14 @@ ComboBox
         implicitWidth: 120
         implicitHeight: 45
         color: control.mainBackgroundColor
-        radius: 20
+        radius: 18
```

Comment:
Revert this change it will break every other combobox

Reply:
All combobox in ui design kit have this radius, 
You think you can create a new combobox with different property?